style.css

```diff
@@ -196,7 +187,7 @@
     font-size: 15px;
     line-height: 24px;
     padding-left: 25px;
-    margin-bottom: 30pxz;
+    margin-bottom: 30;
     margin-top: 5px;
   }
 
```

Comment:
- [ ] Please this card image it not showing so make sure that you fix it to make your work match the Figma design. :pray: 
  
  ![image](https://user-images.githubusercontent.com/92303891/235319276-8353b5dc-ab35-461e-b69f-540685eed38b.png)